media-libs/oyranos/oyranos-9999.ebuild

```diff
@@ -80,18 +88,37 @@ multilib_src_configure() {
 	local libdir=$(get_libdir)
 	local mycmakeargs=(
 		-DLIB_SUFFIX=${libdir#lib}
+		-DUSE_SYSTEM_ELEKTRA=YES
+		-DUSE_SYSTEM_YAJL=YES
+		$(cmake-utils_use_find_package cairo)
+		$(cmake-utils_use_find_package cups)
+		$(cmake-utils_use_find_package doc Doxygen)
+		$(cmake-utils_use_find_package exif Exif2)
+		$(cmake-utils_use_find_package jpeg)
+		$(cmake-utils_use_find_package raw LibRaw)
+		$(cmake-utils_use_find_package tiff)
+		$(cmake-utils_use_find_package X X11)
+	)
 
-		$(usex cairo -DWANT_CAIRO=1 "")
-		$(usex cups -DWANT_CUPS=1 "")
-		$(usex doc -DWANT_HTML=1 "")
-		$(usex exif -DWANT_EXIV2=1 "")
-		$(usex raw -DWANT_LIBRAW=1 "")
-		$(usex X -DWANT_X11=1 "")
+	# prefer Qt5
+	if ! use qt5 ; then
+		use qt4 && mycmakeargs+=( $(cmake-utils_use_use qt4) )
+	fi
 
-		# only used in programs
-		$(multilib_native_usex fltk -DWANT_FLTK=1 "")
-		$(multilib_native_usex qt4 -DWANT_QT4=1 "")
-	)
+	# only used in programs
+	if ! multilib_is_native_abi ; then
+		mycmakeargs+=(
+			-DCMAKE_DISABLE_FIND_PACKAGE_Fltk=ON
+			-DCMAKE_DISABLE_FIND_PACKAGE_Qt4=ON
+			-DCMAKE_DISABLE_FIND_PACKAGE_Qt5=ON
+		)
+	else
+		mycmakeargs+=(
+			$(cmake-utils_use_find_package fltk)
```

Comment:
Again, in the common block:

    -DCMAKE_DISABLE_FIND_PACKAGE_Fltk=$(multilib_native_usex fltk OFF ON)

(note the reverse logic because we won't `ON` on non-multilib)